fastplotlib/graphics/line.py

```diff
@@ -53,3 +56,32 @@ def update_colors(self, colors: np.ndarray):
 
         self.world_object.geometry.colors.data[:] = self.colors
         self.world_object.geometry.colors.update_range()
+
+    @property
+    def indices(self) -> Any:
+        return self.indices
+
+    @property
+    def features(self) -> List[str]:
+        return self.features
+
+    def _set_feature(self, name: str, new_data: Any, indices: Any):
+        if name == "color":
+            self.update_colors(new_data)
+        elif name == "data":
+            self.update_data(new_data)
+        else:
+            raise ValueError("name arg is not a valid feature")
+
+    def link(self, event: str, target: Graphic, feature: str, new_data: Any, indices_mapper: callable = None):
+        valid_events = ["click"]
+        if event in valid_events:
+            self.world_object.add_event_handler(self.event_handler, event)
+            self.events[event] = list()
```

Comment:
This should be an instance variable

Reply:
Should be set in the init of the graphic, not here it'll reset every time it's called